Polyhedron/demo/Polyhedron/Use_ssh.cpp

```diff
@@ -191,7 +191,7 @@ bool push_file(ssh_session &session,
     return false;
   }
   res = ssh_scp_push_file
-      (scp, dest_path, size, 0707);
+      (scp, dest_path, size, 0777);
```

Comment:
And then, for a file, the meaning of `0777` is a bit different, but the security issue is the same: the "write" permission should not be given to everybody. The correct mode is `0644`.